swarm/pss/pubsub.go

```diff
@@ -0,0 +1,41 @@
+package pss
```

Comment:
This doesn't belong in the `pss` package. Let's move it to `pusher`.

Reply:
This totally belongs to the pss package IMHO